README.md

```diff
@@ -25,13 +25,53 @@ The library is under heavy development and is a public preview project under an
 
 ## Installation & Usage
 
-Currently, the library is released in a temporary CLI named `kced`. The latest release of the CLI can be downloaded for your OS from the [releases page](https://github.com/Kong/go-apiops/releases). Downloaded and extract the release archive to install. [Docker images](https://hub.docker.com/r/kong/kced) are also available for the temporary CLI.
+Currently, the functionality is released in a temporary CLI named `kced`. The CLI can be installed locally or ran as a Docker container.
 
-The [Documentation](./docs/README.md) page provides command details and examples. The CLI also provides a `help` command to see usage details:
+### Local Install
 
+* Download the latest release archive of the CLI for your OS from the [releases page](https://github.com/Kong/go-apiops/releases).
+* Once you have downloaded, extract the release archive contents somewhere in your `PATH`, for example:
+
+  To extract:
+  ```
+  tar xvf ~/Downloads/go-apiops_0.1.11_darwin_all.tar.gz -C ~/go/bin
+  ```
+
+  And test the installation:
+  ```
+  ~/go/bin/kced version
+  ```
+
+  Should print the installed version:
+  ```
+  kceD v0.1.11 (347b296)
+  ```
+
+### Docker Install
+
+[Docker images](https://hub.docker.com/r/kong/kced) are available on Docker Hub and can be ran with:
+
+```bash
+docker run kong/kced:v0.1.11 version
```

Comment:
Is there a `:latest` convenience tag?

Reply:
No

Reply:
Might be worth adding one if possible. That way people can build the `docker run kong/kced` muscle memory and automatically opt in as we add new functionality

Reply:
I realize this is a personal subjective opinion, but I am anti-latest tag. We clarify above that these are examples and the user should use the desired version.   Guiding users to use latest, which might end up in their CI/CD system, is my concern and is not best practice.

Reply:
Not a blocker for this release, but as a consumer I'd expect to see a latest tag (even if the docs point to specific versions)

Reply:
Personally, I never use them and don't think they should be provided. 🤷

Reply:
Understood, and disagree. `deck` offers a `:latest` tag (https://hub.docker.com/r/kong/deck/tags), as do most other projects. Not having one would surprise me as a consumer. 

I expect `kong/kced` to have a `:latest` tag for the beta release. Let me know if you'd like me to handle it

Reply:
As stated, it's a personal opinion and one I realize is counter to the norm.  I disagree w/ the usage generally (including `deck`s) and think it's bad practice especially for tools intended for automated workflows. I would support a connivence tag on major semtag versions where we follow breaking change rules similar to how GitHub actions are most commonly used.

Of course, feel free to add `latest` at your discretion.